typescript-selenium/utils/DriverHelper.ts

```diff
@@ -0,0 +1,274 @@
+        }
+
+        throw new Error(`Exceeded maximum typing attempts, to the '${elementLocator}' element`)
+    }
+
+    public async clear(elementLocator: By, timeout = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        const attempts: number = TestConstants.TS_SELENIUM_DEFAULT_ATTEMPTS
+        const polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING
+
+
+        for (let i = 0; i < attempts; i++) {
+            const element: WebElement = await this.waitVisibility(elementLocator, timeout);
+
+            try {
+                await element.clear()
+                return
+            } catch (err) {
+                if (err instanceof error.StaleElementReferenceError) {
+                    await this.wait(polling)
+                    continue;
+                }
+
+                throw err
+            }
+        }
+
+        throw new Error(`Exceeded maximum clearing attempts, to the '${elementLocator}' element`)
+    }
+
+    public async enterValue(elementLocator: By, text: string, timeout = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        await this.waitVisibility(elementLocator, timeout)
+        await this.clear(elementLocator, timeout)
+        await this.waitAttributeValue(elementLocator, "value", "", timeout)
+        await this.type(elementLocator, text, timeout)
+        await this.waitAttributeValue(elementLocator, "value", text, timeout)
+    }
+
+    public async waitAndSwitchToFrame(iframeLocator: By, timeout = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        await this.driver.wait(until.ableToSwitchToFrame(iframeLocator), timeout)
+    }
+
+    public async waitAndGetText(elementLocator: By, timeout = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT): Promise<string> {
+        const attempts: number = TestConstants.TS_SELENIUM_DEFAULT_ATTEMPTS
+        const polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING
+
+
+        for (let i = 0; i < attempts; i++) {
+            const element: WebElement = await this.waitVisibility(elementLocator, timeout);
+
+            try {
+                const innerText: string = await element.getText()
+                return innerText
+            } catch (err) {
+                if (err instanceof error.StaleElementReferenceError) {
+                    await this.wait(polling)
+                    continue;
+                }
+
+                throw err
+            }
+        }
+
+        throw new Error(`Exceeded maximum text obtaining attempts, from the '${elementLocator}' element`)
+    }
+
+    public async waitAndGetValue(elementLocator: By, timeout = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT): Promise<string> {
+        const elementValue: string = await this.waitAndGetElementAttribute(elementLocator, 'value', timeout)
+        return elementValue
+    }
+
+    public async waitUntilTrue(callback: any, timeout = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        await this.driver.wait(callback(), timeout)
+    }
+
+    public async reloadPage() {
```

Comment:
I don't see much value in these one-liner that just delegate to another method with the same signature.